internal/security/config/contract/command.go

```diff
@@ -0,0 +1,21 @@
+//
+// Copyright (c) 2020 Intel Corporation
+//
+// SPDX-License-Identifier: Apache-2.0'
+//
+
+package contract
```

Comment:
standard is the package should be called "interfaces".
https://github.com/edgexfoundry/edgex-go/tree/master/internal/core/data/interfaces
https://github.com/edgexfoundry/edgex-go/tree/master/internal/core/metadata/interfaces
etc.